docs/UsingCppInterOp.rst

```diff
@@ -67,33 +67,204 @@ that it was designed for.
 
 Further Enhancing the Dynamic/Automatic bindings in CPPYY
 =========================================================
+
+CppInterOp ties into the following major enhancements in cppyy:
+
+-  **Reducing dependencies** of cppyy to minimize code bloat and make it
+   faster.
+
+-  A **cppyy-Style Library** (i.e., CppInterOp) that enables interoperability
```

Comment:
doesn't fit with the topic of the list which is how cppyy is enhanced

Reply:
rephrased the topic to "How cppyy relates to CppInterOp and how it works"
(we're trying to fit a cppyy doc into an existing cppinterop doc, but this is the placement that was decided after several discussions)